client/src/modules/payroll/weekend_configuration/templates/action.tmpl.html

```diff
@@ -11,12 +11,6 @@
       </a>
     </li>
 
-    <li>
-      <a data-method="configure" ui-sref="configurationWeekend.config({id : row.entity.id})" href>
```

Comment:
Can we remove the templates/controllers for this?  It seems like we can now delete whatever files make up the `configurationWeekend.config()` state.